<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRTablePlotWidget</class>
 <widget class="QWidget" name="QFRDRTablePlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QVisibleHandleSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="layPlot">
       <item>
        <widget class="JKQtPlotter" name="plotter" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabPlots">
       <attribute name="title">
        <string>&amp;Plots</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="btnAddGraph">
            <property name="text">
             <string>add graph</string>
            </property>
            <property name="icon">
             <iconset resource="qfrdrtable.qrc">
              <normaloff>:/table/plot_add.png</normaloff>:/table/plot_add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnDeleteGraph">
            <property name="text">
             <string>delete graph</string>
            </property>
            <property name="icon">
             <iconset resource="qfrdrtable.qrc">
              <normaloff>:/table/plot_delete.png</normaloff>:/table/plot_delete.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listGraphs"/>
        </item>
        <item>
         <widget class="QWidget" name="widGraphSettings" native="true">
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cmbGraphType">
             <item>
              <property name="text">
               <string>points/line graph</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vertical impulses</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>horizontal impulses</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>filled curve X</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>filles curve Y</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vertical steps</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>horizontal steps</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>vertical bars</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>horizontal bars</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>T&amp;itle:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edtGraphTitle"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>X Data:</string>
             </property>
             <property name="buddy">
              <cstring>cmbLinesXData</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cmbLinesXData"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>X Errors:</string>
             </property>
             <property name="buddy">
              <cstring>cmbLinesXError</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cmbLinesXError"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Y Data:</string>
             </property>
             <property name="buddy">
              <cstring>cmbLinesYData</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Y Errors:</string>
             </property>
             <property name="buddy">
              <cstring>cmbLinesYError</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cmbLinesYData"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="cmbLinesYError"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSystem">
       <attribute name="title">
        <string>&amp;System</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>X axis label:</string>
          </property>
          <property name="buddy">
           <cstring>edtXLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edtXLabel"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Y axis label:</string>
          </property>
          <property name="buddy">
           <cstring>edtYLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="edtYLabel"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edtTitle"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>&amp;title:</string>
          </property>
          <property name="buddy">
           <cstring>edtTitle</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>x axis range:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>y axis range:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>axis scaling:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="chkLogX">
            <property name="text">
             <string>x: log-scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLogY">
            <property name="text">
             <string>y: log-scale</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFDoubleEdit" name="edtXMin"/>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFDoubleEdit" name="edtXMax"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnAutoscaleX">
            <property name="text">
             <string>autoscale</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFDoubleEdit" name="edtYMin"/>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFDoubleEdit" name="edtYMax"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnAutoscaleY">
            <property name="text">
             <string>autoscale</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>key:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="chkShowKey">
            <property name="text">
             <string>show key</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>grid:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="chkGrid">
          <property name="text">
           <string>show grid</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>font name:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QFontComboBox" name="cmbFontname"/>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>key font size:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QDoubleSpinBox" name="spinKeyFontSize">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>256.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>axis tick font size:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QDoubleSpinBox" name="spinAxisFontSize">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>256.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>axis label font size:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QDoubleSpinBox" name="spinAxisLabelFontSize">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>256.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actAddGraph">
   <property name="icon">
    <iconset resource="qfrdrtable.qrc">
     <normaloff>:/table/plot_add.png</normaloff>:/table/plot_add.png</iconset>
   </property>
   <property name="text">
    <string>add graph</string>
   </property>
  </action>
  <action name="actDeleteGraph">
   <property name="icon">
    <iconset resource="qfrdrtable.qrc">
     <normaloff>:/table/plot_delete.png</normaloff>:/table/plot_delete.png</iconset>
   </property>
   <property name="text">
    <string>delete graph</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQtPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qfrdrtable.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>graphDataChanged()</slot>
  <slot>plotDataChanged()</slot>
 </slots>
</ui>
